### ISSUE TITLE
Feature Request - search with Any media only provides results for 'shows cataloged_item'

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7424]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
_A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

See details in https://github.com/ArctosDB/arctos/issues/7422

A search for Any media was not providing results for any records with associated PDFs

Search: https://arctos.database.museum/search.cfm?guid_prefix=UWBM%3AMamm&customoidoper=LIST&media_type=any

was expected to include https://arctos.database.museum/guid/UWBM:Mamm:84151 in results, but does not because the PDF is associated with the "documents cataloged_item" relationship.

**Describe what you're trying to accomplish** 
_A clear and concise overview of the goals; why are you asking for this?_

Find any record with associated media, no matter the "how" of the association.

**Describe the solution you'd like**
_How might we accomplish your goals?_

Make this fact clear or make the Any search also find "documents_catalog_record" or actually any record with linked media? That is what I think people are expecting but not getting and maybe not realizing they are missing things?

**Describe alternatives you've considered**
_A clear and concise description of any alternative solutions or features you've considered._

Dusty says use media search - but that only finds things one relationship at a time and does not provide a list of catnums, but a list of media.

**Additional context**
_Add any other context or screenshots about the feature request here._

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._

**Comment by dustymc** [comment]
closing for lack of interest

**Comment by campmlc** [comment]
I disagree there is lack of interest. This is clearly providing misleading results. I support reopening and resolving, adding to Media committee.

**Comment by wellerjes** [comment]
I agree; can there be an option that searches for any media? We have several botany specimens that just have images of labels attached, but not the images of the specimen itself. It's difficult to get an accurate count when the parameters are just "shows cataloged_item"

**Comment by dustymc** [comment]
Possibly https://github.com/ArctosDB/arctos/issues/6665#issuecomment-2371846590 is some reference to this??

**Comment by dustymc** [comment]
abandoned
